### DISCUSSION TITLE
GitHub releases are failing

### DISCUSSION THREAD

**Discussion by Pesa** [pradyunsg/furo#828] [Potential Issue]
The [last](https://github.com/pradyunsg/furo/actions/runs/10262724329) [two](https://github.com/pradyunsg/furo/actions/runs/9988021273) releases on GitHub failed, seemingly due to a mismatch in the artifact name: [`distributions`](https://github.com/pradyunsg/furo/blob/696ceb13f060dc505053f91ac4d46f0915c261be/.github/workflows/release.yml#L23) (upload) vs. [`python-package-distributions`](https://github.com/pradyunsg/furo/blob/696ceb13f060dc505053f91ac4d46f0915c261be/.github/workflows/release.yml#L78) (download).

**Comment by pradyunsg** [comment]
I'm aware -- those are not issues since those are releases meant to go to TestPyPI. :)

**Reply by Pesa** [reply]
So you're saying that creating a GitHub release [here](https://github.com/pradyunsg/furo/releases) is not the goal of that workflow? That's fair. I assumed it was. Not a big deal since I'll eventually be notified of the new version by a PR from dependabot anyway :)
